.github/workflows/gate.yaml

```diff
@@ -5,6 +5,22 @@ on:
   pull_request:
     branches: [ 'master', 'stabilization*' ]
 jobs:
+  validate-centos7:
+    name: Build, Test on CentOS 7 (Container)
+    runs-on: ubuntu-latest
+    container:
+      image: centos:7
+    steps:
+      - name: Install Deps
+        run: yum install -y cmake make openscap-utils PyYAML libxslt expat xml-common python-jinja2
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Build
+        run: ./build_product rhel7 --derivatives
```

Comment:
We risk missing build issues in other products, we should consider building `rhel8`, `jre` and `firefox` too.

For example, current master has an issue with Rule.to_ocil() and rule `service_systemd-coredump_disabled` when building RHEL8 content on python 2.

Reply:
I just added all the products back.